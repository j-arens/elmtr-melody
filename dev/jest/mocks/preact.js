module.exports = {
    unmountComponent: jest.fn(),
};
