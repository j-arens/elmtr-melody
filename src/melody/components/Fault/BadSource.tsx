import { h } from 'preact';

export default () => (
    <p class="melody__body">
        Unable to play track, the source was not found.
    </p>
);
