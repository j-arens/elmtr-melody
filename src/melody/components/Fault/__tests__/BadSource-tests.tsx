import { h } from 'preact';
import { render } from 'preact-render-to-string';
import BadSource from '../BadSource';

describe('DefaultError', () => {
    it('renders', () => {
        const component = render(<BadSource/>);
        expect(component).toMatchSnapshot();
    });
});
