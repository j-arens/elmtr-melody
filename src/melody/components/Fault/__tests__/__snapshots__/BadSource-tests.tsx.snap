// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultError renders 1`] = `<p class="melody__body">Unable to play track, the source was not found.</p>`;
